core/chains/evm/txmgr/orm.go

```diff
@@ -42,6 +42,60 @@ type orm struct {
 
 var _ ORM = (*orm)(nil)
 
+// Directly maps to database schema.
+// Do not modify type unless you
+// intend to modify the database schema
+type dbReceipt struct {
+	ID               int64
+	TxHash           common.Hash
+	BlockHash        common.Hash
+	BlockNumber      int64
+	TransactionIndex uint
+	Receipt          evmtypes.Receipt
+	CreatedAt        time.Time
+}
+
+// Directly maps to database schema.
```

Comment:
This is NOT a db receipt. This is receipt object, as returned from geth.

Reply:
rename to rawOnchainReceipt. I think that name is more clear.

Reply:
One should interpret the phrase `database schema` to be all-encompassing whether it refers to a table, column, enum, etc.. Since this is stored as a column (jsonb) column in the eth_receipts table one could say it maps to the database schema.

Onchainreceipt sounds good though